molecad/builder.py

```diff
@@ -0,0 +1,70 @@
+# import argparse
+from typing import (
+    Optional,
+    Union
+)
+
+
+def input_specification(
+        domain: str,
+        namespace: str,
+        identifiers: Union[int, str]
+) -> str:
+    """
+    This function builds the first part of URL setting up input identifiers.
+    :param domain: string refers to database - compound | substance | assay
+    :param namespace: string can assign values depending on the domain.
+    #TODO make description
+    if domain == `compound`:
+    if domain == `substance`:
+    if domain == `assay`:
```

Comment:
```suggestion
    if domain == ``compound``:
    if domain == ``substance``:
    if domain == ``assay``:
```

Мы пишем документацию в формате restuctured text. Лучше сразу к нему привыкать